data.sql

```diff
@@ -1,34 +1,87 @@
+VALUES ( 'Agumon', '2020/2/3', 0, TRUE, 10.23);
 
-INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES ('2', 'Gabumon', '2020/11/15', 2, TRUE, 8);
+INSERT INTO animals (name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Gabumon', '2020/11/15', 2, TRUE, 8);
 
-INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES ('3', 'Pikachu', '2021/1/7', 1, FALSE, 15.04);
+INSERT INTO animals (name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Pikachu', '2021/1/7', 1, FALSE, 15.04);
 
-INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES ('4', 'Devimon', '2017/5/12', 5, TRUE, 11);
+INSERT INTO animals ( name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ( 'Devimon', '2017/5/12', 5, TRUE, 11);
 
- INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES ( 5,'Charmander', '2020/2/8', 0, FALSE, 11);
+ INSERT INTO animals ( name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020/2/8', 0, FALSE, 11);
 
- INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES (6,'Plantmon', '2020/11/15', 2, TRUE, 5.7);
+ INSERT INTO animals ( name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2020/11/15', 2, TRUE, 5.7);
 
-INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES (7,'Squirtle', '1993/4/2', 3, FALSE, 12.13);
+INSERT INTO animals ( name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Squirtle', '1993/4/2', 3, FALSE, 12.13);
 
- INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES (8,'Angemon', '2005/6/12', 1, TRUE, 45);
+ INSERT INTO animals ( name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Angemon', '2005/6/12', 1, TRUE, 45);
 
- INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES (9,'Boarmon', '2005/6/7', 7, TRUE, 20);
+ INSERT INTO animals ( name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Boarmon', '2005/6/7', 7, TRUE, 20);
 
-INSERT INTO animals (id, name, date_of_birth, secape_attempts, neutered, weight_kg)
-VALUES (10,'Blossom', '1998/10/13', 3, TRUE, 17);
+INSERT INTO animals ( name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Blossom', '1998/10/13', 3, TRUE, 17);
+
+ INSERT INTO animals (name, date_of_birth, secape_attempts, neutered, weight_kg)
+VALUES ('Ditto', '2022/6/14', 4, TRUE, 22);
+
+
+
+
+-- inserting DATABASE
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners(full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners(full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners(full_name, age) VALUES ('Melody pond', 77);
+
+INSERT INTO owners(full_name, age) VALUES ('Deian watchster', 14);
```

Comment:
- [ ] Here's a typo in this `full_name`. Kindly modify it to the correct value `Dean Winchester`